core/src/main/scala/org/http4s/SecureSession.scala

```diff
@@ -0,0 +1,9 @@
+package org.http4s
```

Comment:
Does this attribute ever make sense on the client, or should we consider putting this in the server package?

Reply:
It probably makes sense to move it the server package unless anyone has other ideas.

Reply:
:+1:

Reply:
Like what @aeffrig did.